examples/cargo_without_cmake/src/main.rs

```diff
@@ -28,6 +32,14 @@ fn main() {
         engine.load(&QUrl::from("qrc:/qt/qml/com/kdab/cxx_qt/demo/qml/main.qml"));
     }
 
+    let str = CString::new("test").unwrap();
+
+    let mut context: QMessageLogContext = QMessageLogContext::default();
+    context.set_line(5);
+    context.set_file(&str);
+
+    qt_message_output(QtMsgType::QtInfoMsg, &context, &QString::from("Test!"));
+
```

Comment:
Accidentally added this, will remove/move later

Reply:
good for testing for now :-)

Reply:
Removed now that it's ready for review